src/formpack/schema/fields.py

```diff
@@ -264,6 +267,20 @@ def get_disaggregated_stats(self, metrics, top_splitters,
     def parse_values(self, raw_values):
         yield raw_values
 
+    @staticmethod
+    def try_get_number(val):
+        try:
+            val = int(val)
+        except ValueError:
+            pass
+        if isinstance(val, int):
```

Comment:
Would `else` be enough (instead of `if isinstance(val, int):`)?

Reply:
Ah beautiful ❤️

```python3
In [1]: def get_num(num):
   ...:     try:
   ...:         num = int(num)
   ...:     except ValueError:
   ...:         pass
   ...:     else:
   ...:         return type(num)
   ...:     try:
   ...:         num = float(num)
   ...:     except ValueError:
   ...:         pass
   ...:     return type(num)
   ...:

In [2]: get_num('42')
Out[2]: int

In [3]: get_num('42.42')
Out[3]: float

In [4]: get_num('42.42a')
Out[4]: str
```